src/__tests__/index.test.ts

```diff
@@ -363,6 +363,11 @@ const vectors: Array<TestVector> = [
       { text: 'AGmV3oHqzfAs3VFiqmn6cecxCXVNyg6tNh', hex: '0ae542fee226c044dc19b036db7cec939777596f' },
     ],
   },
+  {
+    name: 'MWC',
+    coinType: 1088,
```

Comment:
Where did you get the coinType? MWC(aka MimbleWimbleCoin) does not seem to be listed on SLIP 44. 

https://github.com/satoshilabs/slips/blob/master/slip-0044.md